src/Cafe/HW/Latte/Renderer/Renderer.cpp

```diff
@@ -37,6 +37,32 @@ bool Renderer::GetVRAMInfo(int& usageInMB, int& totalInMB) const
 	return false;
 }
 
+
+void Renderer::Initialize()
+{
+	// imgui
+	imguiFontAtlas = new ImFontAtlas();
+	imguiFontAtlas->AddFontDefault();
+
+	auto setupContext = [](ImGuiIO& io){
+		io.WantSaveIniSettings = false;
+		io.IniFilename = nullptr;
+	};
+
+	imguiTVContext = ImGui::CreateContext(imguiFontAtlas);
+	setupContext(ImGui::GetIO());
+	imguiPadContext = ImGui::CreateContext(imguiFontAtlas);
+	setupContext(ImGui::GetIO());
```

Comment:
`ImGui::CreateContext` only changes the current context if there is none set yet. So there should be a `SetCurrentContext` after the second `CreateContext`. I would add it to both because it's not obvious that the context is being set as a side effect of creating it.